### DISCUSSION TITLE
Trying to install php with openssl on OSX 10.9.5 to get TLS 1.2 working

### DISCUSSION THREAD

**Discussion by BaxterZZZ** [Homebrew/discussions#2468] [Everyday usage]
First, I apologize for needing help with an old OS. My office runs on software that that only works on 10.9 and earlier, so we are stuck with that for now. Everything was fine except we use an API to post data to FedEx to get labels and they just started enforceing TLS 1.2 min. The computer has TLS 1.2 support, but php still uses php OpenSSL 0.9.8za even though I was able to install openssl 1.0.2 that supports TLS 1.2 and curl uses it fine. I had trouble installing homebrew, but I was able to install brew via tigerbrew... but nothing seems to work right in terms of downloading and building php, which, as I understand it is the only way to get php to start using openssl 1.0.2 (I'm fine with using 1.1.1 if that will work on my system. I have intel i5 macs from 2010-2014. If someone can get me started, I'll be happy to try things and post error messages.

Thanks! (and desperate for help after spending about 30 hrs trying to google my way through this....)

**Comment by carlocab** [comment]
Have you tried using MacPorts? They have much better support for old versions of macOS.

**Reply by BaxterZZZ** [reply]
I found links online to macports and will try that next if I don't get a solution here or from Tigerbrew. Homebrew was reviewed as being so much more popular and "easier/safer" than macports that I wanted to get it to work if I could. But so far it hasn't been easy for me. Thanks!
